src/main/scala/zio/interop/twitter.scala

```diff
@@ -22,9 +22,9 @@ import zio.{ Runtime, Task, UIO, ZIO }
 
 package object twitter {
   implicit class TaskObjOps(private val obj: Task.type) extends AnyVal {
-    final def fromTwitterFuture[A](future: Task[Future[A]]): Task[A] =
+    final def fromTwitterFuture[A](future: => Future[A]): Task[A] =
```

Comment:
Could you keep the old function and mark it as deprecated?

Reply:
Thanks, will do!

To avoid further rounds of feedback, I have a couple of follow up questions:

1) Should I add a full set of tests for the deprecated entrypoint?

2) The `@deprecated` annotation supports providing a message and a version/date since the deprecation occurred. Does this look good?
`@deprecated("Use fromTwitterFuture[A](future: => Future[A]) instead", "v20.6.0.0-RC2")`

Thanks!

Reply:
No need for copy-paste, you can just define this one as "proxy", as the other one will eventually be removed or made private.

```suggestion
    final def fromTwitterFuture[A](future: => Future[A]): Task[A] = fromTwitterFuture(Task(future))
```

Reply:
Wouldn't that always trigger deprecation warnings? As the deprecated `fromTwitterFuture[A](future: Task[Future[A]])` will always be called as the proxy?

Perhaps separating the implementation into a common one used by both the @deprecated and the new one could avoid that, but I can't think of a _simple_ way to do so.